resources/ext/build-scripts/install-vendored-gems.sh

```diff
@@ -19,5 +19,5 @@ while read LINE
 do
   gem_name=$(echo $LINE |awk '{print $1}')
   gem_version=$(echo $LINE |awk '{print $2}')
-  java -cp puppet-server-release.jar clojure.main -m puppetlabs.puppetserver.cli.gem --config jruby.conf -- install ${gem_name} --version ${gem_version}
+  java -cp puppet-server-release.jar:jruby-1_7.jar clojure.main -m puppetlabs.puppetserver.cli.gem --config jruby.conf -- install ${gem_name} --version ${gem_version}
```

Comment:
this worries me a bit but also I guess it's probably not super bad because this is only installing a specific list of gems, so so long as we don't have any gems that don't work with 1.9.3 in there it's fine?